### ISSUE TITLE
[Website] Connect: Ensures <li> elements are used semantically.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6429]
### Target Platforms

NodeJS

### SDK Version

https://adaptivecards.io/connect/ 9/14/2021

### Application Name

Website

### Problem Description

[ADO 36107873](https://microsoft.visualstudio.com/OS/_workitems/edit/36107873)

**Repro Steps:**
1. Launch the above URL.
2. Now, run the accessibility insights for web.
3. Observe that list items are not used semantically.

**Actual Result:**
List item does not have a `<ul>, <ol>` parent element without a role, or a role="list" i.e. for the breadcrumbs menu 'Adaptive Cards' and for 'Connect' under connect web page.

**Expected Result:**
LI elements should be used semantically, for the breadcrumbs menu 'Adaptive Cards' and for 'Connect' under connect web page.

### Card JSON

```JSON
No card
```

**Comment by PankajBhojwani** [comment]
Closing this as the `adaptivecards.io/connect` page has now been removed
